### PR TITLE
Addition of eMotionTech materials and Strateo3D nozzles sizes

### PR DESCRIPTION
**Pull request by AButler31**
-Addition of new materials for 0.4mm, 0.6mm and 0.8mm nozzles
-Update of existing materials to allow 1.0mm and 1.2mm nozzles

### CODE REVIEW

File: emotiontech_copa.xml.fdm_material [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                <setting key="hardware compatible">yes</setting>
+                <setting key="print temperature">225</setting>
+                <setting key="retraction amount">1.5</setting>
+            </hotend>
+            <hotend id="Standard 1.2">
+                <setting key="hardware compatible">yes</setting>
+                <setting key="print temperature">245</setting>
+                <setting key="retraction amount">1.9</setting>
+            </hotend>
+            -->
```
Comment (konskarm) [inline emotiontech_copa.xml.fdm_material:55]: How come this is commented out? And why are the hotends here not named "Experimental" like in the rest of the materials?

File: emotiontech_pc.xml.fdm_material [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                <setting key="hardware compatible">yes</setting>
+                <setting key="print temperature">225</setting>
+                <setting key="retraction amount">1.5</setting>
+            </hotend>
+            <hotend id="Standard 1.2">
+                <setting key="hardware compatible">yes</setting>
+                <setting key="print temperature">245</setting>
+                <setting key="retraction amount">1.9</setting>
+            </hotend>
+            -->
```
Comment (konskarm) [inline emotiontech_pc.xml.fdm_material:55]: How come this is commented out? And why are the hotends here not named "Experimental" like in the rest of the materials?

File: emotiontech_pla_hr_870.xml.fdm_material [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+            <!-- <hotend id="Standard 1.0"> -->
+                <!-- <setting key="hardware compatible">yes</setting> -->
+                <!-- <setting key="print temperature">225</setting> -->
+                <!-- <setting key="retraction amount">1.5</setting> -->
+            <!-- </hotend> -->
+            <!-- <hotend id="Standard 1.2"> -->
+                <!-- <setting key="hardware compatible">yes</setting> -->
+                <!-- <setting key="print temperature">245</setting> -->
+                <!-- <setting key="retraction amount">1.9</setting> -->
+            <!-- </hotend> -->
```
Comment (konskarm) [inline emotiontech_pla_hr_870.xml.fdm_material:53]: How come this is commented out? And why are the hotends here not named "Experimental" like in the rest of the materials?

File: emotiontech_copa.xml.fdm_material (hunk 1 above)
Comment (ChronosTech) [reply emotiontech_copa.xml.fdm_material:55]: I created a new PR based on this #156 , I add the "Experimental".

In addition, I answered why it was commented out.

File: emotiontech_pc.xml.fdm_material (hunk 2 above)
Comment (ChronosTech) [reply emotiontech_pc.xml.fdm_material:55]: I created a new PR based on this #156 , I add the "Experimental".

In addition, I answered why it was commented out.

File: emotiontech_pla_hr_870.xml.fdm_material (hunk 3 above)
Comment (ChronosTech) [reply emotiontech_pla_hr_870.xml.fdm_material:53]: I created a new PR based on this #156 , I add the "Experimental".

In addition, I answered why it was commented out.

**Comment by Ghostkeeper** [comment]
Closing this one then in favour of https://github.com/Ultimaker/fdm_materials/pull/156.

**Pull request closed without merging**